packaging/irods_prc_tests/test_rule_engine_plugin_metadata_guard_atomic.py

```diff
@@ -0,0 +1,57 @@
+from irods.session import iRODSSession
+from irods.meta import iRODSMeta
+from irods.meta import iRODSMeta, AVUOperation
+
+import irods.exception
+
+import sys
+import unittest
+import tempfile
+import shutil
+import os
+import json
+
+class test_metadata_guard(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        with iRODSSession(host='localhost', port=1247, user='rods', password='rods', zone='tempZone') as session:
+            a = session.users.create_with_password('alice', 'test') 
```

Comment:
Let's replace `a` with a better name.

I know this is a WIP, but I'm highlighting it so we don't forget about it.